src/runtime_src/core/edge/drm/zocl/Makefile

```diff
@@ -3,12 +3,31 @@ KERNEL_SRC ?= /lib/modules/$(shell uname -r)/build
 # Base on Linux Documentation/kbuild/modules.txt.
 # $(src) provides the absolute path by pointing to the directory
 # where the currently executing kbuild file is located.
+ifneq ($(src),)
+# building modules
+ifneq ($(wildcard $(src)/../common),)
+# dkms dir
+common_dir = ../common
+else
+# build from source code
+common_dir = ../../../common/drv
+endif
+
+else
+# clean
+common_dir = ../../../common/drv
+ifeq ($(wildcard ../../../common/drv),)
+common_dir = ../common
+endif
+
+endif
+
 ccflags-y := -I$(src)/include -I$(src)/../../include -I$(src)/../../../include -I$(src)/../../../common/drv/include
 
-drv_common-y   := ../../../common/drv/kds_core.o \
-		  ../../../common/drv/cu_hls.o \
-		  ../../../common/drv/xrt_cu.o \
-		  ../../../common/drv/xrt_xclbin.o
+drv_common-y   := $(common_dir)/kds_core.o \
+		  $(common_dir)/cu_hls.o \
```

Comment:
At least if you configure correctly GitHub you can see all this tabs/spaces cuisine.
This is how I can see it, actually. :-)